packages/components/src/Combobox/index.ts

```diff
@@ -1 +1,3 @@
-export { Combobox } from "./Combobox";
+export { Combobox, ComboboxProps } from "./Combobox";
+
+export { ComboboxContextProvider } from "./ComboboxProvider";
```

Comment:
Finally! The inverse of my nitpicks

```suggestion
export { Combobox, ComboboxProps } from "./Combobox";
export { ComboboxContextProvider } from "./ComboboxProvider";
```

Reply:
no breathing 4 u